src/main/java/pvs/app/controller/ProjectController.java

```diff
@@ -69,6 +69,12 @@ public ProjectController(ProjectService projectService, RepositoryService reposi
         return ResponseEntity.status(HttpStatus.OK).body(successMessage);
     }
 
+    @PatchMapping("/project")
+    public ResponseEntity<String> renameProject(@RequestParam("name") String name, @RequestParam("projectId") Long projectId) {
```

Comment:
Since the path for this API is not pointed out WHAT should be patched inside a project, so

Suggest make these parameters to be an optional parameter for preventing any exceptions.
Another solution is giving them a default value.

```suggestion
    @PatchMapping("/project")
    public ResponseEntity<String> renameProject(@RequestParam(required = false) String name, @RequestParam(required = false) Long projectId) {
```

And also suggest limit this API path scope to `/project/name` for more clear meaning.